aea/helpers/search/models.py

```diff
@@ -27,28 +27,6 @@
 ATTRIBUTE_TYPES = Union[float, str, bool, int]
```

Comment:
OEF Models now live here, the reason being that they are different from protocols (we want to have control over them from the framework side). Them being in OEF protocol also introduces many cross dependencies.